staff/maryam-malek/cv/index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Curriculum Vitae</title>
+   
+</head>
+<body>
+        <header>
+            <h1 id="name">Richard House</h1>
+            <address>
+                <p>Email adress:<br><a href="mailto:webmaster@example.com">webmaster@example.com</a></p>
+                <p>Portafolio<br><a href="mailto:webmaster@example.com">example.com</a></p>
+                <p>Street beta</p>
+                <p>London</p>
+            </address>
+        </header>  
+        <section id="studies">
+            <h2>Studies</h2> 
+                <article>
+                    <h4> Compulsory High School Education</h4>
+                    <aside>
+                        <p>Oxford High School</p>
+                        <p>London</p>
+                    </aside>
+                </article>
+                <article>
+                    <h4>Arts and Humanities</h4>
+                    <aside>
+                        <p>University of London</p>
+                        <p>London</p>
+                    </aside>
+                </article>
+                <article>
+                    <h4>Master in Roman art</h4>
+                    <aside>
+                        <p>University of London</p>
+                        <p>London</p>
+                    </aside>
+                </article>           
+        </section>
+     <section id="experience">
+        <h2>Experience</h2> 
+        <article>
+                <h4> Lorem ipsum dolor sit amet eos?</h4>
+                <aside>
+                    <p>ipsum dolor sit amet eos</p>
+                    <p>London</p>
+                </aside>
+            </article>
+            <article>
```

Comment:
article does not seem to apply correctly here, as this content has more a sense of a section. please, RTFM and correct the code.
- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article
- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section